### ISSUE TITLE
Are we able to set up multiple cameras in the 3DreamEngine?

### ISSUE THREAD

**Issue by SugarRayLua** [3dreamengine/3DreamEngine#59]
It appears from the documentation that we might be able to set up multiple cameras, but it didn't seem clear if the multiple camera feature was yet set up. If it is, it would helpful to know how to switch between cameras to render different points of view (e.g. letting the user choose between two character avatars in the scene). Thanks!

**Comment by Luke100000** [comment]
I added a demo which uses multiple cameras at once. This is more than you wanted but should present how cameras and target canvases can be used, including recursive rendering.

For your specific case you can either:
* Set the default camera directly. Maybe you don't need two cameras to begin with
* Pass the camera as the first argument in present, e.g. `dream:present(yourCamera)`

**Comment by SugarRayLua** [comment]
Thanks, @Luke100000, that looks great and renders well in Love 11.4 on iOS on my iPad!

I'll study that example and work on some projects with different cameras.

Have a good weekend. :-)

**Comment by SugarRayLua** [comment]
@Luke100000, your 3D engine is amazing!

Is there consideration to make incorporate it as Love's 3D engine? If note, is there a formal way to request that it be added to future Love releases? (Not sure how the process works as a novice to the Love community). I'd be happy to submit a request. :-)

**Comment by SugarRayLua** [comment]
(P.S. added 3DreamEngine to the Love wiki page on 3D engines)

**Comment by Luke100000** [comment]
Thanks! Glad you like it. Since 3Dream is pure Lua and works as a otherwise independent library it does not make sense to include it in love itself.

**Comment by SugarRayLua** [comment]
Makes sense :-). Still working through all your examples to understand how to build my own projects. Is there a preferred place to ask questions if not understanding other features or how to set something up using the engine?

**Comment by Luke100000** [comment]
I think the fastest way to reach me is on Discord, 3Dream has a server: https://discord.gg/3MCRPzTzKg

**Comment by SugarRayLua** [comment]
Perfect, thanks!
